### DISCUSSION TITLE
Can't run cypress docker included image in default browser i.e. electron

### DISCUSSION THREAD

**Discussion by monasser112** [cypress-io/cypress#27480] [Questions and Help]
am trying to run the following command 
`docker run -it -v ${pwd}:/e2e -w /e2e cypress/included:latest --browser electron`
but I keep receiving that Browser: electron was not found on your system or is not supported by cypress, though electron browser is the default browser and come baked into cypress.
latest image has major version 12. 

![cypress-docker-image-issue-1](https://github.com/cypress-io/cypress/assets/38355025/e9f6e453-ac5e-45b7-bdef-b7a30be442bd)
[cypress included image ](https://hub.docker.com/r/cypress/included)

**Comment by portyu9** [comment]
Hi @monasser112 

For a `cypress/included` image, Electron was bundled with Cypress and historically was the default browser, so you should not have needed to explicitly pass `--browser electron` just to get the default run.

One thing I would definitely change is this:

```text
cypress/included:latest
```

`latest` is mutable. Cypress recommends pinning an exact image/version so your Cypress, Node, and browser versions do not unexpectedly change underneath the CI run.

For the original issue, try the pinned included image with the default browser first and inspect:

```bash
cypress info
cypress version
```

inside that exact image if Cypress says the browser is unavailable.

For new CI configuration there is another important change: as of Cypress 16, the bundled Electron browser is **deprecated** and will be removed in a future major version. I would therefore migrate the job to an installed supported browser such as Chrome rather than building new automation around Electron.

Please mark this answer as accepted if it helped, thank you.
